artifacts/definitions/Windows/Remediation/Quarantine.yaml

```diff
@@ -0,0 +1,359 @@
+                        if(condition=Stdout, 
+                            then=Stdout, 
+                            else= if(condition=Stderr,
+                                then=Stderr,
+                                else= if(condition= ReturnCode=0, 
+                                    then ='Entry added: ' + 
+                                        join(array=filter(list=cmdline[0].CMD,
+                                            regex='^\\w+=.+'),sep=' ')       
+                            ))) AS Result
+                    FROM 
+                        execve(argv=cmdline[0].CMD,sep='\r\n')
+                    GROUP BY Results
+                })
+      
+      # third step is to create policy actions
+      - LET create_actions = SELECT * FROM foreach(
+                row= { 
+                    SELECT 
+                        Action, 
+                        'TRUE' as FilterAction 
+                    FROM policy 
+                    GROUP BY Action 
+                },
+                query={
+                    SELECT 
+                        timestamp(epoch=now()) as Time,
+                        if(condition=Stdout, 
+                            then=Stdout, 
+                            else= if(condition=Stderr,
+                                then=Stderr,
+                                else= if(condition= ReturnCode=0, 
+                                    then ='FilterAction added: ' + 
+                                        join(array=filter(list=cmdline[0].CMD,
+                                            regex='^\\w+=.+'),sep=' ')       
+                            ))) AS Result
+                    FROM 
+                        execve(argv=cmdline[0].CMD,sep='\r\n')
+                })
+    
+      # fourth step combines action lists and actions in a Rule
+      - LET create_rules = SELECT * FROM foreach(
+                row= { 
+                    SELECT 
+                        Action, 
+                        'TRUE' AS Rule 
+                    FROM policy 
+                    GROUP BY Action 
+                },
+                query={
+                    SELECT 
+                        timestamp(epoch=now()) as Time,
+                        if(condition=Stdout, 
+                            then=Stdout, 
+                            else= if(condition=Stderr,
+                                then=Stderr,
+                                else= if(condition= ReturnCode=0, 
+                                    then ='Rule added: ' + 
+                                        join(array=filter(list=cmdline[0].CMD,
+                                            regex='^\\w+=.+'),sep=' ')       
+                            ))) AS Result
+                    FROM 
+                        execve(argv=cmdline[0].CMD,sep='\r\n')
+                })
+      
+      # fith step is to enable our IPSec policy
+      - LET enable_policy = SELECT * FROM foreach(
+                row=['EnablePolicy'],
```

Comment:
I am not sure why these are in a foreach?

Reply:
For these commands, there are no rows to send so I can not add a field for the switch case. Instead I added a single item array to switch on _value.

The other option is just run the command directly here, but I like the standard of a cmd switch case with all my IPSec cases in one spot.